src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1994,22 +1994,22 @@ void os::init(void) {
 
   Bsd::initialize_system_info();
 
-  // _main_thread points to the thread that created/loaded the JVM.
+  // _main_thread points to the thread that created/loaded the JVM
```

Comment:
If the "sentence" starts with a variable name then you can't capitalize it, so this could have stayed as it was, but is also fine without the period.

Reply:
I treated non-multiline comment as in-line comment, without the need for capitalization and period. I was trying to have some consistency.